src/main/resources/detect-sh.sh

```diff
@@ -156,7 +156,7 @@ get_detect() {
   if [ ${USE_REMOTE} -eq 1 ]; then
     echo "getting ${DETECT_SOURCE} from remote"
     TEMP_DETECT_DESTINATION="${DETECT_DESTINATION}-temp"
-    curlReturn=$(curl ${DETECT_CURL_OPTS} --silent -w "%{http_code}" -L -o "${TEMP_DETECT_DESTINATION}" --create-dirs "${DETECT_SOURCE}")
+    curlReturn=$(curl ${DETECT_CURL_OPTS} --retry --silent -w "%{http_code}" -L -o "${TEMP_DETECT_DESTINATION}" --create-dirs "${DETECT_SOURCE}")
```

Comment:
Retry requires an argument with the number of retries it should perform. The above change will throw an error.

```
curl: option --retry: expected a proper numerical parameter
curl: try 'curl --help' or 'curl --manual' for more information
```

Reply:
Ugh, I mis-copied from my sanity check, one moment